go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/Alluxio/alluxio-csi
+
+go 1.13
+
+require (
+	github.com/container-storage-interface/spec v1.1.0
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/inconshreveable/mousetrap v1.0.0 // indirect
+	github.com/kubernetes-csi/csi-lib-utils v0.2.0 // indirect
+	github.com/kubernetes-csi/drivers v1.0.2
+	github.com/spf13/cobra v0.0.3
+	github.com/spf13/pflag v1.0.3 // indirect
+	golang.org/x/net v0.0.0-20190514140710-3ec191127204
+	golang.org/x/sys v0.0.0-20190516110030-61b9204099cb // indirect
+	golang.org/x/text v0.3.2 // indirect
+	google.golang.org/appengine v1.4.0 // indirect
+	google.golang.org/genproto v0.0.0-20190516172635-bb713bdc0e52 // indirect
+	google.golang.org/grpc v1.20.1
+	k8s.io/apimachinery v0.0.0-20190515023456-b74e4c97951f // indirect
+	k8s.io/kubernetes v1.14.2
```

Comment:
@madanadit Should we stick to K8s v1.16 for CSI as well?

Reply:
@mingfang is this the minimum version required? can the same csi artifact be deployed to multiple version of k8s?